### DISCUSSION TITLE
getShopByOwnerUserId - different handling for draft stores

### DISCUSSION THREAD

**Discussion by assaflei** [etsy/open-api#770] [Ideas]
When using this API call ([getShopByOwnerUserId](https://developers.etsy.com/documentation/reference/#operation/getShopByOwnerUserId)) on draft stores (I would also like to get a link to an article explaining this status) I will receive this error message:
`Could not find a shop for user with user_id = 123456789`

Is there an option to get a more elaboration on the messages returned by the API so I could better inform our customers about the setup required from them?

**Comment by etsyachristensen** [comment]
@assaflei You are correct. That error could be a bit better worded. This endpoint will only return shops if they are active. A shop owner may have a shop that is still not active (just signed up but haven't added payment info etc), on vacation or frozen/banned.
